packages/perseus-editor/src/widgets/expression-editor.tsx

```diff
@@ -38,11 +37,12 @@ type DefaultProps = {
 };
 
 const parseAnswerKey = ({key}: AnswerForm): number => {
-    const parsedKey = Number.parseInt(key ?? "");
```

Comment:
I have no memory of this code and it doesn't look like my style. I rarely try/catch unless I am trying to _prevent_ errors from being thrown.

I'm not going to dig into git blame, but this code means nothing to me and I don't think we should be throwing here.